python-checks/src/test/resources/checks/hotspots/graphQLIntrospection.py

```diff
@@ -0,0 +1,227 @@
+
+def flask_graphql_non_compliant(schema, some_middleware, introspection_middleware):
+    from flask_graphql import GraphQLView
+    from middleware import IntrospectionMiddleware, get_introspection_middleware
+
+    GraphQLView.as_view(  # Noncompliant {{Disable introspection on this GraphQL server endpoint.}}
+#   ^[el=+5;ec=5]
```

Comment:
Are we sure we want to highlight the full call? This may be noisy (Not asking for a change, though - just asking).

Reply:
I was not sure the ticket said the method `as_view`. I guess it would be enough to highlight the name only